components/bl-mapbox/src/helpers.js

```diff
@@ -199,7 +199,7 @@ export const useMarkers = (markers, mapRef, onMarkerClick) => {
         popup.on('open', () => {
           const coordinates = { lat: markerItem.coordinates.lat, lng: markerItem.coordinates.lng };
 
-          onMarkerClick({ coordinates, description: description || '' });
+          onMarkerClick({ coordinates, description: description || '', markerData });
```

Comment:
Why do we need to do this with `coordinates`?

Can we get them from markerItem and pass here?